honeybee_radiance_folder/folder.py

```diff
@@ -500,6 +501,56 @@ def dynamic_scene(self, indoor=False, reload=False):
         return self._dynamic_scene_indoor if indoor \
             else self._dynamic_scene
 
+    def grid_info(self, info_type_is_model=False):
```

Comment:
Since this input is for this function I think you can make the name of the argument smaller and save the full description for the description itself. I would use `is_model` or `model` instead of `info_type_is_model`.

```suggestion
    def grid_info(self, is_model=False):
```